.github/workflows/javascript.yml

```diff
@@ -12,6 +12,7 @@ on:
 
 jobs:
   lint:
+    name: Lint Javascript
```

Comment:
`JavaScript V3`? (@cpyle0819)

Reply:
we don't use the SDK version anywhere else. Should that be added to all the names?

Reply:
I'd say that's an easy "no" lol